project/build.properties

```diff
@@ -1,2 +1,2 @@
-sbt.version=0.13.9
+sbt.version=0.13.10
```

Comment:
Isn't there an 0.13.16?

Reply:
There's 0.13.17 infact, but bumping to it introduced a bunch of deprecation warnings and I didn't want to do lots of SBT refactoring so went more conservative

Reply:
That's fair.  This project might get folded into http4s, if standalone blaze isn't going to do websockets anymore (paging @bryce-anderson), so let's not fight that.